lib/std/bounded_array.zig

```diff
@@ -39,14 +39,17 @@ pub fn BoundedArrayAligned(
 ) type {
     return struct {
         const Self = @This();
+        /// Smallest integer type that is able to hold values from 0 to `buffer_capacity`.
+        const Len = std.meta.Int(.unsigned, std.math.log2_int(u16, buffer_capacity) + 1);
```

Comment:
```suggestion
        const Len = std.math.IntFittingRange(0, buffer_capacity);
```

Reply:
Thanks, I looked for something like that in std.meta, but not std.math for some reason